src/csharp/Microsoft.Spark/Utils/AssemblyLoader.cs

```diff
@@ -90,6 +95,11 @@ internal static class AssemblyLoader
         /// found.</exception>
         internal static Assembly LoadAssembly(string assemblyName, string assemblyFileName)
         {
+            if (string.IsNullOrWhiteSpace(assemblyFileName))
```

Comment:
This condition is present when we serialize UDF in the repl.

Reply:
Can you add the comments to the code?